sanity-check/main.py

```diff
@@ -192,13 +193,15 @@ def check_analysis_exists(metadata,submission_song_url):
                 analysis["experiment"]["experimental_strategy"]==metadata.get('experimental_strategy') and \
                 analysis['analysisType']['name']=="sequencing_experiment"\
                 :
-                    sys.exit(
-                        "Sample '%s'/'%s' has an existing published analysis '%s' for experiment_strategy '%s.'"
+                    analysis_exists=True
+                    error_message=\
```

Comment:
error_message=error_message+";"+

Reply:
do we consider to concatenate the error messages for all published analysis?

Reply:
Added:
https://github.com/icgc-argo/argo-data-submission/blob/6c00aa1620fcc8d8ea9a89900ac9482a9f30ddc4/sanity-check/main.py#L202-L206

Expect multiple `PUBLISHED` analyses to be an edge cases. Displaying a subset should suffice?